README.md

```diff
@@ -476,6 +486,25 @@ precached resources.
 _Default_: `false`
 
 
+## Wrappers and Starter Kits
+
+While it's possible to use the `sw-precache` module's API directly within any
+JavaScript environment, several wrappers have been developed by members of the
+community tailored to specific build environments. They include:
+- [`sw-precache-webpack-plugin`](https://www.npmjs.com/package/sw-precache-webpack-plugin)
+- [`sw-precache-brunch`](https://www.npmjs.com/package/sw-precache-brunch)
+- [`grunt-sw-precache`](https://www.npmjs.com/package/grunt-sw-precache)
+- [`exhibit-builder-sw-precache`](https://www.npmjs.com/package/exhibit-builder-sw-precache)
+
```

Comment:
Yes! I had a reminder to suggest we link up to wrappers but I see the hive mind is alive, well and in full force ;) Nice work linking these up. I was familiar with the first two of these but hadn't seen the latter. 

Perhaps my only suggestion here mid-long term is keeping in touch with the authors of these wrappers and gently pinging them (maybe even for feedback) when a major API change is underway. I've found that to be a useful way to encourage contributions from folks maintaining wrappers too.